clients/admin-ui/src/features/data-discovery-and-detection/action-center/components/InProgressMonitorTasksList.tsx

```diff
@@ -59,33 +59,34 @@ export const InProgressMonitorTasksList = () => {
   );
 
   const filterContent = (
-    <div className="min-w-[220px] space-y-3 p-2">
-      <Checkbox.Group
-        value={statusFilters}
-        onChange={handleStatusesChanged}
-        className="flex flex-col gap-2"
-      >
-        {availableStatuses.map((status) => (
-          <Checkbox key={status} value={status}>
-            {formatStatusForDisplay(status)}
-          </Checkbox>
-        ))}
-      </Checkbox.Group>
-      <Checkbox
-        checked={showDismissed}
-        onChange={(e) => updateShowDismissed(e.target.checked)}
-        className="mb-2"
-      >
-        Show dismissed tasks
-      </Checkbox>
-      <Space>
+    <div className="min-w-[220px] p-2">
+      <div className="mb-5 flex flex-col gap-1.5">
```

Comment:
Should avoid `div`'s if possible
```suggestion
      <Flex vertical className="mb-5 gap-1.5">
```